Generate content of the knowledge base index page (#794)
Avaiga/taipy-doc: docs/knowledge_base/demos/fraud_detection.md

---
title: Fraud Detection
category: demos
type: code
data-keywords: gui dashboard vizelement layout chart ai multi-page classification

> short-description: A Taipy Application that analyzes credit card transactions to detect fraud.

Reviewer: ```suggestion
short-description: Fraud detection application based on credit card transactions analyses.
```

img: images/fraud_threshold.png
---

> A Taipy Application that analyzes credit card transactions to detect fraud.

Reviewer: ```suggestion
An application that analyzes credit card transactions to detect fraud.
```
